test/exprtest/CMakeLists.txt

```diff
@@ -28,9 +28,15 @@ set(SYCLBLAS_EXPRTEST ${CMAKE_CURRENT_SOURCE_DIR})
 set(SYCL_EXPRTEST_SRCS
   ${SYCLBLAS_EXPRTEST}/blas1_scal_asum_test.cpp
   ${SYCLBLAS_EXPRTEST}/blas1_axpy_copy_test.cpp
-  ${SYCLBLAS_EXPRTEST}/collapse_nested_tuple.cpp
 )
 
+# Temporary disabling the following tests for Intel DPC++ as currently Intel compiler crashes while running the following tests
```

Comment:
I think it would be good to mention the llvm ticket in this comment and/or track the issue somewhere on Jira. Worried that we might forget to re-enable the test after the llvm bug is fixed.

Reply:
Added the link to the bug